### ISSUE TITLE
classDiagram new line in class label strange behaviour

### ISSUE THREAD

**Issue by guerinaxel** [mermaid-js/mermaid#4665]
### Description

Hello,

When using a classDiagram, if you set a label with a new line ("\n"), the following content of the class will be messed up...

### Steps to reproduce

1. Go to mermaid.live
2. Create a class diagram with a class having at least "\n" in its label
3. Add anything in your class (property, functions...)

### Screenshots

![image](https://github.com/mermaid-js/mermaid/assets/7128887/695b40b2-28ea-40ca-a4b5-0285332fa0aa)

### Code Sample

```text
// From mermaid.live
classDiagram
class B["label:\nreturn"] {
        + a
        + b
        c
        d()
    }
```

### Setup

- Mermaid version: mermaid.live
- Browser and Version: Chrome 114, Brave 1.51.114

**Comment by ibrahimWassouf** [comment]
This has just recently been fixed in #4633. I'm assuming that since the live editor displays a version, it means that it does not follow the develop branch but rather the newest release version. You should see it get fixed in the next update if I'm not mistaken.

**Comment by Yokozuna59** [comment]
As @ibrahimWassouf said, you'll be able to the changes with next release.

Thanks @ibrahimWassouf!
